src/lexer.coffee

```diff
@@ -1008,7 +1008,7 @@ exports.Lexer = class Lexer
           lastToken[2].range[1]
         ]
       ]
-      lparen[2] = lparen.origin[2]
+      lparen[2] = lparen.origin[2] unless quote?.length
```

Comment:
I traced this back [here](https://github.com/jashkenas/coffeescript/pull/4572/files#r130409263) to your comments PR

There I think you were running into `lparen` sometimes not having *any* location data set

But it generally does, and this line was wiping out the accurate location data (eg for the opening `///` in an interpolated regex) and replacing it with the location data for the entire interpolated string/regex

So updated it to only "overwrite" `lparen`'s location data if it doesn't already have legitimate location data

Reply:
Always nice to have my own bugs pointed out and fixed 😄